views/pages/GraphPage.jsx

```diff
@@ -173,6 +151,7 @@ const GraphPage = () => {
   }
 
   React.useEffect(() => {
+    setOpenSidebar(!openSidebar)
```

Comment:
why are we toggling the sidebar when this mounts?

Reply:
The original functionality is such that when a user is in the matrix page they only see the matrix page without the sidebar.

Reply:
but in this case if they go to the page when the sidebar is closed then it will be opened, right?

Reply:
why not explicitly set this to `setOpenSidebar(false)` instead of toggling?